binding-library/java/src/main/java/com/microsoft/azure/functions/signalr/annotation/SignalROutput.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.functions.signalr.annotation;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.ElementType;
+
+import com.microsoft.azure.functions.annotation.CustomBinding;
+
+/**
+ * <p>Output type to Azure SignalR Service</p>
+ */
+@Target({ ElementType.PARAMETER, ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@CustomBinding(direction = "out", name = "signalRMessage", type = "SignalR")
```

Comment:
Should `name` have a value? In @pragnagopa's sample it is an empty string: https://github.com/Azure/azure-functions-java-worker/pull/271#issue-252885238

Reply:
do you suggest we assign "$return" in CustomBinding, which requires user to use it as Method?

---
In reply to: [259248651](https://github.com/Azure/azure-functions-signalrservice-extension/pull/48#discussion_r259248651) [](ancestors = 259248651)

Reply:
Perhaps making it required (no default) to match the other bindings in the library is better.

Reply:
Sure. :)

---
In reply to: [259256163](https://github.com/Azure/azure-functions-signalrservice-extension/pull/48#discussion_r259256163) [](ancestors = 259256163)